packages/world/src/modules/keyswithvalue/KeysWithValueHook.sol

```diff
@@ -16,15 +16,14 @@ import { getTargetTableSelector } from "./getTargetTableSelector.sol";
  * We can optimize this by adding support for `setIndexOfField` in Store
  * and then replicate logic from solecs's Set.sol.
  * (See https://github.com/latticexyz/mud/issues/444)
+ *
+ * Note: if a table with composite keys is used, only the first key is indexed
```

Comment:
Even though it doesn’t throw now, does this help with the issue @yonadaaa had? Is there a quick way to support composite keys? (ie hashing)

Reply:
> Even though it doesn’t throw now, does this help with the issue @yonadaaa had?

Can you elaborate which issue you're referring to?